api/author_api.py

```diff
@@ -0,0 +1,35 @@
+import flask
+from flask import Blueprint
+
+from messages.author_messages import NewAuthorRequest, UpdateAuthorRequest
+from services.author_service import AuthorService
+
+author_api = Blueprint('user_api', __name__)
```

Comment:
`author_api = Blueprint('user_api', __name__)`
Not sure what the implications are if you are using `user_api` as the blueprint name for the `author_api`. Probably none, just wanted to note.